lib/dolly/collection.rb

```diff
@@ -1,6 +1,7 @@
 module Dolly
   class Collection
     extend Forwardable
+    include Enumerable
```

Comment:
including enumerable will remove the need to use forwardable and forward the methods like `+` or `-` to the collection object.... this is a different approach... originally all the enum methods where kept away on purpose.... so more refactor is required if we will do this